### PR TITLE
Separate out @deephaven/console component

### PR DESCRIPTION
**Pull request by mofojed**
- Includes everything in the Console and CommandHistory components
- Also includes StorageTable - that should probably be factored out into a separate package as well (@deephaven/storage?)

### CODE REVIEW

File: packages/console/src/common/ObjectIcon.tsx [hunk 1]
```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { dhTable, vsGraph, vsPreview } from '@deephaven/icons';
 import dh from '@deephaven/jsapi-shim';
-import { ReactComponent as PandasIcon } from '../../assets/svg/pandas_mark_white_tiny.svg';
```
Comment (vbabich) [inline packages/console/src/common/ObjectIcon.tsx:5]: If we don't need the Pandas icon, maybe delete the svg too, to not keep the unused asset in the codebase.

**Review by vbabich — DISMISSED** [review]

File: packages/console/src/common/ObjectIcon.tsx (hunk 1 above)
Comment (dsmmcken) [reply packages/console/src/common/ObjectIcon.tsx:5]: Why are we dropping the pandas icon?

Comment (mofojed) [reply packages/console/src/common/ObjectIcon.tsx:5]: It should be put into @deephaven/icons instead #37 created

Comment (dsmmcken) [reply packages/console/src/common/ObjectIcon.tsx:5]: Can't, it's multi-colored.

**Review by vbabich — APPROVED** [review]

File: packages/console/src/common/ObjectIcon.tsx (hunk 1 above)
Comment (dsmmcken) [reply packages/console/src/common/ObjectIcon.tsx:5]: This merged with no resolution, and no follow up tickets.

Comment (mofojed) [reply packages/console/src/common/ObjectIcon.tsx:5]: It'll be captured in #37 . If we can't add it to @deephaven/icons, then need to add direct to @deephaven/console, and have `svgr` somewhere in the pipeline.

**Pull request merged**